test/artillery/artillery-test-users-id-tasks-id-api.yaml

```diff
@@ -0,0 +1,23 @@
+config:
+  target: "https://nsc-fun-dev-usw2-thursday.azurewebsites.net/api"
+  phases:
+    - duration: 10
+      arrivalRate: 4
+      name: Warm up 1 request per second
+
+    # # Testing at higher load  
+    # - duration: 1200
+    #   arrivalRate: 2
+    #   rampTo: 6
+    #   name: Ramp up load 2-6 requests / second
+    # - duration: 1800
+    #   arrivalRate: 10
+    #   name: Sustained load 10 Requests per second
+
```

Comment:
Same thing here, just deleting outdated code, but the YAML file looks fine as well